### ISSUE TITLE
How to test traininng model？

### ISSUE THREAD

**Issue by Finger-tiao** [rapidrabbit76/SketchColorization#2]
1. If possible could you release code for test the training model?
2. I follow your code to train on other dataset of 20,000 images. But both draft and colorization results are bad. It seems that both stages failed to learn the color of the hint point? 
<img width="508" alt="屏幕快照 2021-06-04 下午4 28 11" src="https://user-images.githubusercontent.com/32674239/120772509-f7c7f900-c552-11eb-91ab-47d0e98a31cf.png">
this image is draft result which train 18 epochs.
Do you have any idea about it? Thanks.

**Comment by rapidrabbit76** [comment]
1. pre-trained models([onnx](https://github.com/rapidrabbit76/SketchColorization/releases/download/1.0.0/SketchColorizationModel.onnx)) are uploaded. for testing, we used the same [Dataset object](https://github.com/rapidrabbit76/SketchColorization/blob/main/data/dataset.py#L161) we used for training.

2. we used 700,000 images. May I know the batch_size, lr and weight of the loss function used for training(Draft Model)?

**Comment by Finger-tiao** [comment]
Thanks for your reply.
The batch size, lr and weight I set are the same as your code. I didn't change your code just use other dataset. By the way, the result of` AutoEncoder` is good.
Before your reply I used my draft model to train on colorization stage. The results are also bad. And I also try to write scripts to test my trained model `SketchColorization.zip`, the results are worser.

This Image is the `fake_image` output by the colorization stage.
<img width="518" alt="屏幕快照 2021-06-08 下午8 07 51" src="https://user-images.githubusercontent.com/32674239/121182156-62e83700-c895-11eb-800b-eb898735c1e6.png">

**Comment by rapidrabbit76** [comment]
thanks for reply.
I check the photos uploaded, it seems that there is no problem with the line art data. There seems to be no problem with the colorization stage.
It seems that you need to check the hint data input in the draft stage.

draft stage receives two hints: line (grayscale) and hint (rgba).

hint data, the rgb channel has scale of -1 to 1,
a(alpha) channel in hint data will have scale of 0 to 1.
I think to check the 128 size hint data of the draft stage.
or check hint data interpolration process

ps. If you send me line art and color image for test, I will test it

**Comment by Finger-tiao** [comment]
Well I'll check the hint data. I see your pre-trained model but how to test it without running app? 
I download your app for mac but it crash back and cannot get in. My mac version is 10.13.6.
